server/store.go

```diff
@@ -30,11 +30,15 @@ type StoreInfo struct {
 	LeaderCount     int
 	RegionCount     int
 	LastHeartbeatTS time.Time
+	LeaderWeight    float64
+	RegionWeight    float64
 }
 
 func newStoreInfo(store *metapb.Store) *StoreInfo {
 	return &StoreInfo{
-		Store: store,
+		Store:        store,
+		LeaderWeight: 1.0,
```

Comment:
How about set default value as 100?

Reply:
If the weight is unset, the score will be the same as old score, because _newScore = oldScore / weight_.